Makefile

```diff
@@ -12,7 +12,8 @@ endif
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.0.1
+VERSION ?= 0.1.0
+TAG_NAME ?= latest
```

Comment:
how we will use `lates` tag? Is it latest released? Or next development version? If latter, shouldn't we use `next`?

Reply:
tbh I'm not yet sure how it will work. it's par of originally generated code.
why do you proposed `next` I saw it here https://github.com/redhat-appstudio/application-service/blob/main/Makefile#L7 
Should I change it to `next`?

Reply:
I don't know. As I understand it, `latest` is usually used for latest released version so if you use it, you will be on latest stable. `next` is used for cutting edge dev branch, that would mean latest commit in `master` branch. So it depends what we want to communicate with this name.